keyboards/efreet/rules.mk

```
@@ -14,11 +14,11 @@ BOOTLOADER = atmel-dfu
 # Build Options
 #   change yes to no to disable
 #
-BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration(+1000)
+BOOTMAGIC_ENABLE = lite      # Virtual DIP switch configuration(+1000)
```

While you're here, would you mind removing these:
```suggestion
BOOTMAGIC_ENABLE = lite      # Virtual DIP switch configuration
```
And the others that GitHub won't let me add suggestions to as it's outside the hunk.

---

Fixed